howto/setup-state-store/setup-couchbase.md

```diff
@@ -60,5 +60,4 @@ kubectl apply -f couchbase.yaml
 
 ### Running locally
 
-The Dapr CLI will automatically create a directory named `components` in your current working directory with a Redis component.
-To use Couchbase, replace the redis.yaml file with the couchbase.yaml above.
```

Comment:
Is this no longer needed?

Reply:
@yaron2 is the Couchbase line no longer needed in the docs?